content/rest-api/filter.md

```diff
@@ -160,6 +160,139 @@ To get the products that were updated during the last 4 days, you can use the fo
 /api/rest/v1/products?search={"updated":[{"operator":"SINCE LAST N DAYS","value":4}]}
 ```
 
+## Filter on product model properties
```

Comment:
specify the version of the PIM (from 2.3)

Reply:
"Filter on product model properties (since v2.3)"